.storybook/.babelrc

```diff
@@ -1 +1,12 @@
-../.babelrc
```

Comment:
What is the point of this? iirc this is here to fix a problem with circleCI

Reply:
That's from when we encountered the issue of running it locally on my PC. It doesn't let you run `yarn storybook` without error unless you were to change that file. We can discard that change.